cfgov/unprocessed/apps/paying-for-college/js/views/school-view.js

```diff
@@ -83,30 +127,9 @@ const schoolView = {
     const prop = input.getAttribute( 'name' );
     const value = input.value;
     updateState.byProperty( prop, value );
+    schoolView._updateProgramList();
 
-    schoolView._checkRadioButtonCount();
-
-  },
-
-  /* Add all event listeners for school search view  */
-  _addListeners: () => {
-    const searchEvents = {
-      keyup: schoolView._handleInputChange
-    };
-    bindEvent( schoolView._searchBox, searchEvents );
-
-    const searchResultsEvent = {
-      click: schoolView._handleResultButtonClick
-    };
-    bindEvent( schoolView._searchResults, searchResultsEvent );
-
-    const radioEvents = {
-      click: schoolView._handleProgramRadioClick
-    };
-    schoolView._programRadios.forEach( elem => {
-      bindEvent( elem, radioEvents );
-    } );
-
+    // schoolView._checkRadioButtonCount();
```

Comment:
Should this be deleted?

Reply:
Yeah, this was a bug I was still working on, but it can all be kiboshed for now.

Reply:
(We now have a variable number of radio buttons for users, depending on their situation, so the whole `_checkRadioButtonCount` method has to be reconsidered. I'm just removing it all for now.)